### PR TITLE
PLAT-7977: add ability to keep socket connected but not listen on specific queue

### PR DESCRIPTION
**Pull request by noam-arad**

### CODE REVIEW

File: lib/ConnectionManager.js [hunk 1]
```diff
@@ -290,6 +291,11 @@ class ConnectionManager {
             throw new Error(KalturaPushServerErrors.TOKEN_DECRYPTION_ERROR);
         }
     };
+
+    _onClientNotListening(socket, eventNameStr, queueKeyStr) {
+        socket.leave(queueKeyStr);
```
Comment (gotlieb) [inline lib/ConnectionManager.js]: what will happen if there is no matching queue to leaver?

Comment (gotlieb) [reply lib/ConnectionManager.js]: if we leave the same room twice?

File: lib/ConnectionManager.js [hunk 2]
```diff
@@ -290,6 +291,11 @@ class ConnectionManager {
...
+        KalturaLogger.log('User is now NOT listening to ' + queueKeyStr);
```
Comment (gotlieb) [inline lib/ConnectionManager.js]: maybe a different message (which user is leaving or more info that we have)

Comment (gotlieb) [reply lib/ConnectionManager.js]: also maybe move function to where all _onX are located for better code reading.

**Comment by yossipapi** [comment]
Noam you did not update the app versions to create the new tag

**Comment by yossipapi** [comment]
Never mind saw you did it in a different pull request

**Pull request merged**